File: SwapNumbers.java
import java.util.Scanner;

public class SwapNumbers {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter first number (a): ");
        int a = sc.nextInt();

        System.out.print("Enter second number (b): ");
        int b = sc.nextInt();

        System.out.println("\nBefore Swap: a = " + a + ", b = " + b);

        // Swapping using third variable
        int temp = a;
        a = b;
        b = temp;

        System.out.println("After Swap: a = " + a + ", b = " + b);
    }
}

File: SwapNumbers.c
#include <stdio.h>

int main() {
    int a, b, temp;

    printf("Enter first number (a): ");
    scanf("%d", &a);

    printf("Enter second number (b): ");
    scanf("%d", &b);

    printf("\nBefore Swap: a = %d, b = %d\n", a, b);

    // Swap using temp variable
    temp = a;
    a = b;
    b = temp;

    printf("After Swap: a = %d, b = %d", a, b);

    return 0;
}
